src/anndata/_io/specs/registry.py

```diff
@@ -405,7 +405,9 @@ def read_elem(elem: StorageType) -> RWAble:
 
 
 def read_elem_lazy(
-    elem: StorageType, chunks: tuple[int, ...] | None = None, **kwargs
+    elem: StorageType,
+    chunks: tuple[int, ...] | None = None,
+    **kwargs,
```

Comment:
This is not an argument here because otherwise it would be passed down to all registered methods, which would fail because we explicitly check `if kwarg not in read_params`